src/applications/medical-copays/components/HTMLStatementList.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useLocation, Link } from 'react-router-dom';
+import moment from 'moment';
+import { sortStatementsByDate } from '../utils/helpers';
+
+const HTMLStatementList = () => {
+  const { pathname } = useLocation();
+  const selectedId = pathname.replace('/balance-details/', '');
+  const statements = useSelector(({ mcp }) => mcp.statements) ?? [];
+  // get selected statement
+  const [selectedCopay] = statements?.filter(({ id }) => id === selectedId);
+  // get facility  number on selected statement
+  const facilityNumber = selectedCopay?.pSFacilityNum;
+  // filter out all statements that are not related to this facility
+  const facilityCopays = statements?.filter(
```

Comment:
The filter and sorts are good places for test (unit and/or cypress

Reply:
Going to write up a ticket to make unit tests for these! Thank you!